x-pack/plugins/spaces/server/saved_objects/mappings.ts

```diff
@@ -19,18 +19,6 @@ export const SpacesSavedObjectMappings = deepFreeze({
         },
       },
     },
```

Comment:
__question:__ any reason we still keep `name` and `_reserved` in the mappings? Are they used in search (server-side) somehow?

Reply:
name.keyword seems to be a sort field, when I try to move it to `schemas` I get the following error:
`{"statusCode":400,"error":"Bad Request","message":"Unknown sort field name.keyword"}`

Unless I was just building the schema wrong; I tried `name: schema.string(),` and `name: schema.object({ keyword: schema.string() }),`, is there a better way?

`_reserved` is moved ++